tools/oesign/oesign.c

```diff
@@ -174,6 +178,50 @@ static int _load_config_file(const char* path, ConfigFileOptions* options)
 
             options->security_version = n;
         }
+        else if (strcmp(str_ptr(&lhs), "IsvFamilyID_H") == 0)
+        {
+            uint64_t n;
+
+            if (str_u64(&rhs, &n) != 0)
```

Comment:
See updated numerical checks, these should check that the values provided are non-negative.